testthisout/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
I'm fine with you trying other things in code but please do not include them in your pull requests moving forward.